test/remote/gateways/remote_cyber_source_test.rb

```diff
@@ -753,6 +753,16 @@ def test_successful_capture_with_issuer_additional_data
     assert !response.authorization.blank?
   end
 
+  def test_successful_capture_with_merchant_reference_number_override
```

Comment:
While i think this override is straightforward and should do what we need, for the sake of being thorough I think it would be good to add a test that demonstrates that the order ID set from the previous authorization still goes through as expected, since there's not existing test coverage for that behavior as far as I can tell 🧐